docs/references/uvh5_memo.tex

```diff
@@ -123,12 +123,20 @@ \subsection{Required Parameters}
 \item \textbf{Nants\_telescope}: \textit{int} The number of antennas in the
   array. May be larger than the number of antennas with data corresponding to
   them. (\textit{Nants\_telescope})
-\item \textbf{ant\_1\_array}: \textit{int} An array of the first antenna indices
-  corresponding to baselines present in the data. This is a one-dimensional
+\item \textbf{ant\_1\_array}: \textit{int} An array of the first antenna numbers
+  corresponding to baselines present in the data. All entries in this array must exist
+  in the antenna\_numbers array. This is a one-dimensional
   array of size Nblts. (\textit{ant\_1\_array})
-\item \textbf{ant\_2\_array}: \textit{int} An array of the second antenna indices
-  corresponding to baselines present in the data. This is a one-dimensional
+\item \textbf{ant\_2\_array}: \textit{int} An array of the second antenna numbers
+  corresponding to baselines present in the data. All entries in this array must exist
+  in the antenna\_numbers array. This is a one-dimensional
   array of size Nblts. (\textit{ant\_2\_array})
+\item \textbf{antenna\_numbers}: \textit{int} An array of the numbers of the antennas
+  present in the array (note that these are not indices, they do not need to start at zero
```

Comment:
Instead of "present in the array" it might be better to say "present in the radio telescope array" (or "comprising the radio telescope array").  There are at least two context sensitive usages of the word "array" here so adding a little verbosity to distinguish from a "UVData array" would be helpful (IMHO).  Same comment applies to `antenna_names` as well.

Reply:
sure, I'll make that change.